content/installation/java/package/maven-cargo.md

```diff
@@ -0,0 +1,193 @@
+-->
+
+##  About the Maven Project
+
+The following instructions walks a Java developer through the necessary steps to add the Contrast Java agent to an existing Maven project and reuse the project's existing integration tests to find vulnerabilities. This includes guiding changes to an [example servlet project](https://github.com/Contrast-Security-OSS/contrast-java-examples/maven-cargo). A completed version of the project [exists in a GitHub branch](https://github.com/Contrast-Security-OSS/contrast-java-examples/tree/maven-cargo-complete/maven-cargo).
+
+## Before You Start
+
+1. Install [Java 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+1. Install [git](https://git-scm.com/).
+1. Verify your [Contrast account](https://app.contrastsecurity.com) connection information.
+
+## Step 1: Gather Contrast account information
+
+The Contrast agent requires some configuration to communicate with the Contrast UI. You'll need to obtain these [four properties](admin-orgsettings.html#apikey) from the Contrast UI:
+
+* The Contrast URL
+* Your Contrast API key
+* The agent username for your Contrast organization
+* The Service key for your Contrast organization's agent user
+
+The Contrast URL is *https://app.contrastsecurity.com/Contrast*, or the URL of your on-premises or private cloud instance. You can find the Contrast URL, your API key and agent username by going to the **user menu > Organization Settings > API tab** in the Contrast UI. You can find your Service key using the link in the **API** tab, or by navigating to the **user menu > Your Account > My Keys section**. 
+
+## Step 2: Create a Contrast configuration file
+
+The Contrast configuration file is a [YAML file](https://yaml.org/). Since it contains API credentials, Contrast recommends you create it in a location that only you have access to. For most developers, your home directory is a good choice. Choose a file name that will make sense to you (e.g., *.contrast.yml*).
+
+* Create the file and open it in your preferred text editor. 
+
+* Paste in the following contents:
+
+```yaml
+api:
+  url: <contrast_url>
+  api_key: <your_api_key>
+  user_name: <agent_user_name>
+  service_key: <agent_user_service_key>
+```
+
+* Replace `<contrast_url>`, `<your_api_key>`, `<agent_user_name>` and `<agent_user_service_key>` with the values you
+obtained from the Contrast UI in the previous step, and save the file.
+
+<details><summary><b>Configuration note</b></summary>
+<p>You can <a href="installation-javaconfig.html">configure the Contrast Java agent</a> using a file, Java system properties or environment variables. For developers, Contrast recommends putting only the variables that are shared across multiple applications (e.g., credentials and Contrast UI connection details) in this
+file. That way you can manage them in one place. <b>Step 9</b> shows you how to set application-specific configurations.</p></details>
```

Comment:
@gilday Was Step 9 removed?

Reply:
Good catch: it was removed when we cut down a lot of steps to simplify the article. I just removed that sentence.